aea/cli/config.py

```diff
@@ -54,89 +63,269 @@ def get(ctx: Context, json_path: List[str]):
     type=click.Choice(["str", "int", "bool", "float"]),
     help="Specify the type of the value.",
 )
-@click.argument("JSON_PATH", required=True, type=AEAJsonPathType())
+@click.argument("JSON_PATH", required=True)
 @click.argument("VALUE", required=True, type=str)
 @pass_ctx
 def set_command(
     ctx: Context,
-    json_path: List[str],
+    json_path: str,
     value: str,
     type: str,  # pylint: disable=redefined-builtin
 ):
     """Set a field."""
-    _set_config(ctx, json_path, value, type)
+    ConfigGetSet(ctx, json_path).set(value, type)
+
 
+class ConfigGetSet:
+    """Tool to get/set value in agent config."""
 
-def _get_config_value(ctx: Context, json_path: List[str]):
-    config_loader = cast(ConfigLoader, ctx.config.get("configuration_loader"))
-    configuration_file_path = cast(str, ctx.config.get("configuration_file_path"))
+    FIELDS_NOT_ALLOWED_TO_CHANGE = [
```

Comment:
See previous comments; should be moved to config!

Reply:
moved

Reply:
Done